Withings/IWithingsClient.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Withings.API.Portable
+{
+    public interface IWithingsClient
+    {
+
+        // go over interface directions and understand meaning before just plugging in
```

Comment:
The interface should mirror all the methods that the eventual WithingsClient has. This is a longer conversation of why this is important, but just know that in our case we like intreface classes because they allow other code that consumes the WithingsClient to write unite tests against the interface, not needing to actually call the Withings API.